--- ssl.conf.orig       2023-04-07 01:28:44.000000000 +0900
+++ ssl.conf    2023-05-14 23:09:00.178143010 +0900
@@ -40,13 +40,13 @@
 <VirtualHost _default_:443>
 
 # General setup for the virtual host, inherited from global configuration
-#DocumentRoot "/var/www/html"
-#ServerName www.example.com:443
+DocumentRoot "/var/www/html"
+ServerName _:443
 
 # Use separate log files for the SSL virtual host; note that LogLevel
 # is not inherited from httpd.conf.
-ErrorLog logs/ssl_error_log
-TransferLog logs/ssl_access_log
+ErrorLog /dev/stdout
+TransferLog /dev/stdout
 LogLevel warn
 
 #   SSL Engine Switch:
@@ -56,7 +56,7 @@
 #   List the protocol versions which clients are allowed to connect with.
 #   The OpenSSL system profile is used by default.  See
 #   update-crypto-policies(8) for more details.
-#SSLProtocol all -SSLv3
+SSLProtocol -all +TLSv1.2
 #SSLProxyProtocol all -SSLv3
 
 #   User agents such as web browsers are not configured for the user's
@@ -189,15 +189,21 @@
 #   Similarly, one has to force some clients to use HTTP/1.0 to workaround
 #   their broken HTTP/1.1 implementation. Use variables "downgrade-1.0" and
 #   "force-response-1.0" for this.
-BrowserMatch "MSIE [2-5]" \
-         nokeepalive ssl-unclean-shutdown \
-         downgrade-1.0 force-response-1.0
+#BrowserMatch "MSIE [2-5]" \
+#         nokeepalive ssl-unclean-shutdown \
+#         downgrade-1.0 force-response-1.0
 
 #   Per-Server Logging:
 #   The home of a custom SSL log file. Use this when you want a
 #   compact non-error SSL logfile on a virtual host basis.
-CustomLog logs/ssl_request_log \
+CustomLog /dev/stdout \
           "%t %h %{SSL_PROTOCOL}x %{SSL_CIPHER}x \"%r\" %b"
 
+  <Directory "/var/www/html/">
+    Options None
+    AllowOverride None
+    Require all granted
+  </Directory>
+
 </VirtualHost>
